apps/verification-portal/src/lib/entities/InboxNotification/Notification.svelte

```diff
@@ -27,10 +33,21 @@
 
     return type ? type : undefined;
   }
+  async function markAsRead() {
+    isPending = true;
+    await markNotificationsAsRead([notification.id])
+      .catch((e) => {
+        console.log(e);
+      })
+      .finally(() => {
+        isPending = false;
+        notification.isRead = true;
+      });
+  }
```

Comment:
Is it not a feature according to FSD?
https://feature-sliced.design/docs/get-started/overview